test/evergreen.yml

```diff
@@ -2659,6 +2677,23 @@ buildvariants:
     - name: ".stress-test-3"
     - name: ".stress-test-4"
 
+- name: performance-tests
+  display_name: Ubuntu 18.04 Performance tests
+  run_on:
+  - ubuntu1804-test
+  expansions:
+    smp_command: -j $(grep -c ^processor /proc/cpuinfo)
+    make_command: PATH=/opt/mongodbtoolchain/v3/bin:$PATH make
+    test_env_vars:
+      LD_PRELOAD=/usr/lib/x86_64-linux-gnu/libeatmydata.so LD_LIBRARY_PATH=$(pwd)/.libs
+      PATH=/opt/mongodbtoolchain/v3/bin:$PATH top_srcdir=$(pwd)/.. top_builddir=$(pwd)
```

Comment:
The `test_env_vars` setting here seems not used by the task included in this build variant, and can be removed?